app/views/articles/_form.html.erb

```diff
@@ -1,26 +1,31 @@
-<% if @article.errors.any? %>
-  <h2>The following errors prevented the article from being saved</h2>
-  <ul>
-    <% @article.errors.full_messages.each do |message| %>
-      <li><%= message %></li>
-    <% end %>
-  </ul>
-<% end %>
-
 <%# Need to provide form with the scope of the Model and the URL to submit %>
-<%# local:true specifies that we want to send via HTTP POST and not remotely via AJAX %>
+<%# local:true specifies that we want to send via HTTP POST and not remotely via
```

Comment:
question: Why change this to 2 lines?

Reply:
If I remember correctly, this was auto formatted by rubocop. I was toggling between off and on the settings for vscode on formatting it for `.html.erb` files as it sometimes provides quite unexpected results.